smartsim/_core/entrypoints/file_operations.py

```diff
@@ -0,0 +1,279 @@
+    else:
+        os.remove(parsed_args.to_remove)
+
+
+def copy(parsed_args: argparse.Namespace) -> None:
+    """Copy the contents from the source file into the dest file.
+    If source is a directory, copy the entire directory tree source to dest.
+
+    Sample usage:
+        python _core/entrypoints/file_operations.py copy
+        /absolute/file/source/path /absolute/file/dest/path
+
+        /absolute/file/source/path: Path to directory, or path to file to
+            copy to a new location
+        /absolute/file/dest/path: Path to destination directory or path to
+            destination file
+    """
+    if not os.path.isabs(parsed_args.source):
+        raise ValueError(f"path {parsed_args.source} must be absolute")
+
+    if not os.path.isabs(parsed_args.dest):
+        raise ValueError(f"path {parsed_args.dest} must be absolute")
+
+    if os.path.isdir(parsed_args.source):
+        dir_util.copy_tree(parsed_args.source, parsed_args.dest)
+    else:
+        shutil.copyfile(parsed_args.source, parsed_args.dest)
+
+
+def symlink(parsed_args: argparse.Namespace) -> None:
+    """
+    Create a symbolic link pointing to the exisiting source file
+    named link.
+
+    Sample usage:
+        python _core/entrypoints/file_operations.py symlink
+        /absolute/file/source/path /absolute/file/dest/path
+
+        /absolute/file/source/path: the exisiting source path
+        /absolute/file/dest/path: target name where the symlink will be created.
+    """
+    if not os.path.isabs(parsed_args.source):
+        raise ValueError(f"path {parsed_args.source} must be absolute")
+
+    if not os.path.isabs(parsed_args.dest):
+        raise ValueError(f"path {parsed_args.dest} must be absolute")
+
+    os.symlink(parsed_args.source, parsed_args.dest)
+
+
+def configure(parsed_args: argparse.Namespace) -> None:
+    """Set, search and replace the tagged parameters for the
+    configure operation within tagged files attached to an entity.
+
+    User-formatted files can be attached using the `configure` argument.
+    These files will be modified during ``Application`` generation to replace
+    tagged sections in the user-formatted files with values from the `params`
+    initializer argument used during ``Application`` creation:
+
+    Sample usage:
+        python _core/entrypoints/file_operations.py configure
+        /absolute/file/source/pat /absolute/file/dest/path tag_deliminator param_dict
+
+        /absolute/file/source/path: The tagged files the search and replace operations
+        to be performed upon
+        /absolute/file/dest/path: The destination for configured files to be
+        written to.
+        tag_delimiter: tag for the configure operation to search for, defaults to
+            semi-colon e.g. ";"
+        param_dict: A dict of parameter names and values set for the file
+
+    """
+    if not os.path.isabs(parsed_args.source):
+        raise ValueError(f"path {parsed_args.source} must be absolute")
```

Comment:
would like to meet with regarding this if you have time!

Reply:
Still willing to meet, but it looks like this has since been resolved! I'm going to close this conversation for now, but feel free to reopen if you feel there is still more to discuss here!!